tests/positive/files/file.go

```diff
@@ -255,3 +257,80 @@ func ForceFileCreationNoOverwrite() types.Test {
 		Config: config,
 	}
 }
+
+func AppendToAFile() types.Test {
+	name := "Append to a file"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	config := `{
+	  "ignition": { "version": "2.2.0-experimental" },
+	  "storage": {
+	    "files": [{
+	      "filesystem": "root",
+	      "path": "/foo/bar",
+	      "contents": { "source": "data:,example%20file%0A" },
+	      "user": {"id": 500},
+	      "group": {"id": 500}
+	    },{
+	      "filesystem": "root",
+	      "path": "/foo/bar",
+	      "contents": { "source": "data:,hello%20world%0A" },
+	      "group": {"id": 0},
+	      "append": true
+	    }]
+	  }
+	}`
+	out[0].Partitions.AddFiles("ROOT", []types.File{
+		{
+			Node: types.Node{
+				Name:      "bar",
+				Directory: "foo",
+				User:      500,
+				Group:     0,
+			},
+			Contents: "example file\nhello world\n",
+		},
+	})
+
+	return types.Test{
+		Name:   name,
+		In:     in,
+		Out:    out,
+		Config: config,
+	}
+}
+
+func AppendToAnEmptyFile() types.Test {
+	name := "Append to an empty file"
```

Comment:
Empty file implies that the file exists but is empty, might make more sense to change to something like `Append to a non-existent file`

Reply:
Done.